docs/content/doc/help/seek-help.en-us.md

```diff
@@ -20,10 +20,25 @@ menu:
 
 **NOTE:** When asking for support, it may be a good idea to have the following available so that the person helping has all the info they need:
 
-1. Your `app.ini` (with any sensitive data scrubbed as necessary)
-2. The `gitea.log` (and any other appropriate log files for the situation)
-    * e.g. If the error is related to the database, the `xorm.log` might be helpful
-3. Any error messages you are seeing
+1. Your `app.ini` (with any sensitive data scrubbed as necessary).
+2. The Gitea logs, and any other appropriate log files for the situation.
+    * The logs may be outputted to console by default. If you need to collect logs from files, 
+      you could copy the following config into your `app.ini` (remove all other `[log]` sections),
+      then you can find the `gitea.log` in Gitea's log directory (default: `%(GITEA_WORK_DIR)/log`).
+    ```
+    ; To show all SQLs, you could also set LOG_SQL=true in [database] section 
```

Comment:
```suggestion
    ```ini
    ; To show all SQL logs, you can also set LOG_SQL=true in the [database] section 
```